src/methods/date.js

```diff
@@ -165,11 +165,12 @@ export function LocalTime(realm: Realm, t: number): number {
 }
 
 // ECMA262 20.3.1.10
-export function UTC(realm: Realm, t: number | Value): NumberValue {
+export function UTC(realm: Realm, _t: number | Value): NumberValue {
+  let t = _t;
```

Comment:
It seems nicer to put a type annotation on t and to modify the assignment below to have an else case.